docs/README.md

```diff
@@ -2,7 +2,7 @@
 
 The purpose of these docs are oriented around NucliaDB development.
 
-If you are getting started using NucliaDB, refer to [getting started](https://docs.nuclia.dev/docs/nucliadb/intro),
+If you are getting started using NucliaDB, refer to [getting started](https://docs.nuclia.dev/docs/docs/nucliadb/intro),
```

Comment:
Isn't a bit redundant this duplicated /docs/docs?

Reply:
yes, it's due to the new doc structure